pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -17,7 +17,7 @@
 import tes
```

Comment:
It's difficult to parse the changes. Can you explain what you changed in this module and why? It doesn't seem obvious to me that this module needed to be changed, or at least not that extensively.

Reply:
Previously, we were trying to submit the task on the single provided TES instance, while now we have a list of ranked TES instances so we have to try for each in order,
Things that I have changed are:
1. Placed the submission of task code inside a for loop, where it tries to submit the task on the instance in the list at the 0th index, otherwise retries with the next instances.
2. Updating the TES instance in the database where the task got submitted.

Reply:
Thanks, that's fine. Was just wondering why there were so many changes, but guess it's because of changes in formatting.

I'm not too happy with catching multiple errors across different calls, but this isn't something you introduced here, so let's leave it like that :)